sql/parser/sql.y

```diff
@@ -1003,13 +1005,19 @@ set_reset_clause:
   SET set_rest {}
 
 show_stmt:
-  SHOW var_name
+  SHOW var_name on_privilege_target_clause for_grantee_clause
```

Comment:
Hmmm, this doesn't look correct. We only want `on_privilege_target_clause` and `for_grantee_clause` for `SHOW GRANTS`. Maybe it's time to figure out how to use a `%prec` specifier to fix this.